x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -562,6 +564,47 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     return await this.baseClient.removeReferencesTo(type, id, options);
   }
 
+  public async openPointInTimeForType(
+    type: string | string[],
+    options: SavedObjectsOpenPointInTimeOptions
```

Comment:
🤔 We may need to add support here for multiple spaces. Since `find` allows you to search across multiple spaces, and `find` supports PITS, it follows that PITS would need this as well.

However, the current usage of PITS in this PR doesn't require multi-space searching, so it's probably OK to defer this body of work. What I would like to see though, is validation within the `find` operation to ensure that you're not specifying a `PIT` with more than one `namespace`. There is a [similar check](https://github.com/elastic/kibana/blob/255779e4ab2c53764be2ea02c51f827fe858f793/x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts#L219-L227) that we can use as a model.

There isn't a real attack vector here, but I would rather prevent these two options from being used together until we've had the time to give this more thought and write proper API tests to verify they work properly.

Reply:
👍  Makes sense, adding this.